charts/victoria-logs-single/Chart.lock

```diff
@@ -4,6 +4,6 @@ dependencies:
   version: 0.46.7
 - name: victoria-metrics-common
   repository: file://../victoria-metrics-common
```

Comment:
Based on an issue related to this PR it seems like this link should also be updated (in `Chart.yaml`) to point to `https://victoriametrics.github.io/helm-charts/`, otherwise an error user is seeing will be the same. Could you update the link?

Reply:
added patching before release